heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -330,34 +314,12 @@ export class JestPlugin implements IHeftPlugin<IJestPluginOptions> {
         '$.moduleLoader': jestResolveMetadata,
         '$.prettierPath': jestResolveMetadata,
         '$.resolver': jestResolveMetadata,
-        '$.runner': JestPlugin._getJsonPathMetadata({
```

Comment:
These are safe to remove? Did Jest change its convention here?

Reply:
It's a mix of cleanup and a change on the Jest side.

The args that were being provided that were different than what the normal `metadata` provided were unused. The other part of what's changed is that we're now looking at the property name to determine which type of resolution we use (either the generic Jest resolution, or the `jest-resolve`-provided resolution functions.